terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf

```diff
@@ -201,28 +200,24 @@ module "server_containerized" {
 
 }
 
-// WORKAROUND: We do the initial testing without a proxy and add it later
-// This will also use a new upcoming proxy_containerized module
-// See https://github.com/SUSE/spacewalk/issues/19280
-// module "proxy" {
-//   source             = "./modules/proxy"
-//   base_configuration = module.base_core.configuration
-//   product_version    = "head"
-//   name               = "pxy"
-//   provider_settings = {
-//     mac                = "aa:b2:92:42:00:a2"
-//     memory             = 4096
-//   }
-//   auto_register             = false
-//   auto_connect_to_master    = false
-//   download_private_ssl_key  = false
-//   install_proxy_pattern     = false
-//   auto_configure            = false
-//   generate_bootstrap_script = false
-//   publish_private_ssl_key   = false
-//   use_os_released_updates   = true
-//   ssh_key_path              = "./salt/controller/id_rsa.pub"
-// }
+module "proxy_containerized" {
+  source             = "./modules/proxy_containerized"
+  base_configuration = module.base_core.configuration
+  product_version    = "head"
+  name               = "pxy"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:a2"
+    memory             = 4096
+  }
+  server_configuration = {
+    hostname = "suma-bv-50-srv.mgr.suse.de"
+    username = "admin"
+    password = "admin"
```

Comment:
Not a fan of such a things in publicly available projects, can we at least encrypt it somehow and decrypt in the moment of authorization?

Reply:
This is not a public repository.
Only SUSE employees have access to it.

Reply:
Are you sure? I was able to access this comment from Incognito mode being signed out.

Reply:
> Not a fan of such a things in publicly available projects, can we at least encrypt it somehow and decrypt in the moment of authorization?
This was simply copy-pasted from the Uyuni/4.3 BV file.